test/Microsoft.OpenAPI.OData.Reader.Tests/Vocabulary/Capabilities/DeprecatedRevisionTypeTests.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Edm.Vocabularies;
+using Microsoft.OpenApi.OData.Vocabulary.Core;
+using Xunit;
+
+namespace Microsoft.OpenApi.OData.Reader.Vocabulary.Capabilities.Tests;
```

Comment:
the namespace looks have different pattern comparing to the existing?

Reply:
if it were up to me, I'd have converted all the existing files to this new format, you get more real-estate on the screen. But doing so might generate tons of conflicts since we have multiple people contributing at this time.
So my compromise was to do it only for new files, so they are future proof, and maybe later we can have a PR that updates the existing files once we're done with our batch of modifications.